include/private/cocoa/CocoaWindow.h

```diff
@@ -56,7 +58,9 @@ namespace lsp
                     CocoaCairoView      *pCocoaView;                    // The View of the window
                     NSCursor            *pCocoaCursor;                  // The Cursor of the View
                     NSWindow            *transientParent;
-                    void place_above(NSWindow *parent);
+                    void                place_above(NSWindow *parent);
+                    std::vector<id>     windowObserverTokens; 
```

Comment:
Looks like it can be replaced by a fixed-size array with predefined indices with some `enum`.